web/src/main/java/model/Account.java

```diff
@@ -0,0 +1,56 @@
+package model;
+
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Data
```

Comment:
@Data from Lombok provides you with the boilerplate code for getters and setters, so you won't need the ones [line 33-56]. Remove them and see how it works.

Reply:
Also, it is much better to put explicitly what you use, instead of using @ Data and not be aware of what it imports. I would suggest: Remove @ Data, add @ Getter, @ Setter, @ RequiredArgsConstructor and remove the already existing code.